src/itdelatrisu/opsu/audio/MultiClip.java

```diff
@@ -186,10 +195,17 @@ private Clip getClip() throws LineUnavailableException {
 		} else {
 			// create a new clip
 			// NOTE: AudioSystem.getClip() doesn't work on some Linux setups.
+			Log.debug("creating a new clip");
 			DataLine.Info info = new DataLine.Info(Clip.class, format);
 			c = (Clip) AudioSystem.getLine(info);
 			if (format != null)
 				c.open(format, audioData, 0, audioData.length);
+			// This is a little hacky, but we can't do an instanceof check and
+			// there's no reason to add the listener unless the system is using
+			// PulseAudioClip et al.
+			if (c.getClass().getSimpleName().equals("PulseAudioClip")) {
+				c.addLineListener(new PulseAudioFixerListener(c));

```

Comment:
I'd prefer if you inline the listener instead of defining a new class for it.

Also, it'd be helpful to comment on any specific version numbers (at least for your system) where this is applicable.